application/positionmanager/src/positionmanager/clients.py

```
@@ -1,6 +1,7 @@
 import requests
 import polars as pl
 from typing import Dict, Any
+import pyarrow as pa
```

Pylint (3.3.7)

[error] Unable to import 'pyarrow'

(E0401)